### PR TITLE
Added email to owner table and data

### PR DESCRIPTION
**Pull request by Ursaaaaaa**
- Before performance audit:

> EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
EXPLAIN ANALYZE SELECT * FROM visits where vets_id = 2;
EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animals_id = 4;

image

image

image

- After Optimization

> CREATE INDEX visits_animal_id_idx ON visits(animals_id);
CREATE INDEX visits_vet_id_idx ON visits(vets_id);
CREATE INDEX owners_email_idx ON owners(email);

image

image

image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -73,4 +73,10 @@ insert into specializations (species_id, vets_id) values ((SELECT id FROM specie
...
+
+
+--performance data
+
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
```
Comment (Mre55) [inline data.sql:82]: Kindly, make sure that after introducing index the performance should be improved and the execution time should be less than 1000ms. Please implement proper index for this `SELECT \* FROM visits where vet_id = 2;` query to achieve the execution time less than 1000ms. I noticed it's above 1000ms in your screenshot.

**Review by Mre55 — CHANGES_REQUESTED** [review]
Hi @ramrod433,

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 

## Required Changes ♻️🛠️ 

- [x] This task is pair-programming and I see your commit only. Kindly, make sure the change has to be implemented by your partner to make commits even.

_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Ursaaaaaa) [reply data.sql:82]: Implemented

**Review by aimemalaika — APPROVED** [review]
Hi @ramrod433  & @aimalamiri , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**